### DISCUSSION TITLE
Common Q&A

### DISCUSSION THREAD

**Discussion by x87** [sannybuilder/dev#111] [Q&A]
Drop your question here, let's find out an answer!

**Comment by MatiDragon-YT** [comment]
Will Sanny Builder ever be on ~~Android~~ others SO?

**Reply by x87** [reply]
Unlikely. Why?

**Reply by MatiDragon-YT** [reply]
Mmm... I don't know, maybe many would be more comfortable using SB natively than emulated by third-party programs.

**Reply by x87** [reply]
it would've made sense, but the percentage of people running (and modding) on those OSes is low.

I'm actually evaluating different ways of making Sanny cross-plaftorm, and updating/writing new functionality in Rust so eventually it can be compiled to any target platform. Also a web-based IDE is another interesting option. I would love to know what are people' thoughts here.

**Comment by MikuAuahDark** [comment]
Why there's no portable/installer-less version?

**Comment by JaggerJam69** [comment]
Will be possible in future for Sanny Builder to create imgui dialogs?

**Reply by x87** [reply]
You can use this plugin https://github.com/user-grinch/CLEOImGui
there is also a proposal to add it to default Sanny config https://github.com/sannybuilder/data/pull/13 I haven't looked yet.
